utilities/mtv_migration.py

```
@@ -31,17 +30,54 @@
+        storage_map = StorageMap(
+            client=plan.client,
+            name=storage_map_name,
+            namespace=storage_map_namespace,
+        )
+
+        # Check if any mapping has offloadPlugin
+        for mapping in storage_map.instance.spec.get("map", []):
+            if "offloadPlugin" in mapping:
+                return True
+
+        return False
```

Ruff (0.15.18)

[warning] Consider moving this statement to an `else` block

(TRY300)